rockcraft/commands/init.py

```diff
@@ -132,8 +132,8 @@ class InitCommand(AppCommand):
 
                 # uncomment this section to enable the async workers for Gunicorn.
                 #   flask-framework/async-dependencies:
-                #     plugin: python
-                #     source: .
+                #       python-packages:
+                #       - gunicorn[gevent]
```

Comment:
Is the spacing correct here?

Reply:
It worked but it is not a good exercise I guess .